src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+  const calculatorObject = {
+    result: 0,
+
+    add(param) {
+      calculatorObject.result += param;
```

Comment:
```suggestion
      this.result += param;
```

Reply:
Code doesn't work with "this", for methods "add", "subtract", "multiply", and "divide". When method "operate" calls some of them, it became "this", but it hasn't own property "result". I think so. For "operate" and "reset" "this" works, thank you.